dts/bindings/gpio/sparkfun-pro-micro-header.yaml

```diff
@@ -16,8 +16,8 @@ description: |
 
     This binding provides a nexus mapping for 18 pins, as depicted below:
 
-        0 TX0                    RAW    -
-        1 RX1                    GND    -
+        1 TX0                    RAW    -
+        0 RX1                    GND    -
```

Comment:
if there's two versions of the number it may be a good idea to make the description here reflect that

Reply:
See comment below. I think this is the only variant in Zephyr.

Reply:
Cool, then maybe add a comment after the table here describing that the first two pins are swapped on some boards and that this binding is used for both variants?

Reply:
Ok I see what you are saying now, sorry https://cdn.sparkfun.com/assets/e/2/7/6/b/ProMicroRP2040_Graphical_Datasheet.pdf got me really confused about the whole story.

Reply:
Actually, could you add a link to https://cdn.sparkfun.com/assets/f/d/8/0/d/ProMicro16MHzv2.pdf in this file maybe? Just for convenience, doubt that url will go anywhere anyway.

Reply:
There, fixed a few minor things (inconsistent use of both Dx/Ay and only Dx, as well as a typo where TX0 became TX0, and RXI became RX1), and added the link.